main.tf

```diff
@@ -804,34 +805,64 @@ module "virtual_lab_manager" {
 }
 
 module "launch_server" {
-  source = "./launch_server"
+  # TODO: remove after the deployment for deletion
+  source     = "./launch_server"
+  aws_region = local.aws_region
+}
+
+module "launch_system" {
+  source = "./launch_system"
+
+  count = var.is_staging ? 1 : 0
 
   aws_region                    = local.aws_region
   vpc_id                        = local.vpc_id
+  account_id                    = local.account_id
   private_alb_listener_arn      = local.private_alb_https_listener_arn
   internet_access_route_id      = local.route_table_private_subnets_id
```

Comment:
This is somewhat confusing: the ID for the internet access route is the ID of the route table for the private subnets? Shouldn't that be `local.route_table_public_id`?

Reply:
I'm mostly judging based on the name of the variable and didn't really dig into what it's being used for. If it's not for public access, then renaming it might make it clearer. I guess this is a legacy artifact that was once intended for public use and then changed :-)

Reply:
Clarified offline, `internet_access_route_id` has probably been named in this way because the private networks can connect to the outside internet thanks to this route table.